src/org/parosproxy/paros/Constant.java

```diff
@@ -346,12 +349,39 @@ public static String getDefaultHomeDirectory(boolean incDevOption) {
         return zapStd;
     		
     }
+    
+    public void copyDefaultConfigs(File f, boolean forceReset) throws IOException, ConfigurationException {
+        FileCopier copier = new FileCopier();
+        File oldf;
+        if (isDevBuild() || isDailyBuild()) {
+            // try standard location
+            oldf = new File (getDefaultHomeDirectory(false) + FILE_SEPARATOR + FILE_CONFIG_NAME);
+        } else {
+            // try old location
+            oldf = new File (zapHome + FILE_SEPARATOR + "zap" + FILE_SEPARATOR + FILE_CONFIG_NAME);
+        }
+        
+        if (!forceReset || oldf.exists() && Paths.get(zapHome).equals(Paths.get(getDefaultHomeDirectory(true)))) {

```

Comment:
This condition is not correct it should be and not or (as the old file needs to exist anyway).
The test was failing because of this, was able to reproduce locally when the default home directory didn't exist (as is the case of Travis CI builds).